src/librustc_unicode/u_str.rs

```diff
@@ -27,9 +27,16 @@ use core::str::Split;
 
 use tables::grapheme::GraphemeCat;
 
-/// An iterator over the words of a string, separated by a sequence of whitespace
-#[stable(feature = "rust1", since = "1.0.0")]
-pub struct Words<'a> {
+#[deprecated(reason = "struct Words is being replaced by struct SplitWhitespace",
+             since = "1.0.0")]
```

Comment:
@nagisa right, but we can always cherry-pick a fix into the beta branch to make this unstable.